gammapy/utils/metadata.py

```diff
@@ -52,8 +52,9 @@ def from_header(cls, hdr):
     def to_yaml(self):
         """Dumps metadata content to yaml."""
         meta = json.loads(self.json())
+        metadict = {"metadata": meta}
```

Comment:
I wonder if a single level is enough here. Or if we need a general `metadata` level with hierarchical entries:
e.g.  ["metadata"]["creation"]